### PR TITLE
[Server-Timing] Header Parsing Tests

### PR DESCRIPTION
**Pull request by cvazac**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/22539)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/326970810)

**Review by yoavweiss — APPROVED** [review]
LGTM!

File: server-timing/resources/parsing/51.js.sub.headers [hunk 1]
```diff
@@ -0,0 +1 @@
+Server-Timing: metric;DuR=123.4;DeSc=description
```
[the remark this answers is not in the document]
Comment (cvazac) [reply server-timing/resources/parsing/51.js.sub.headers]: Good idea! Added:
https://github.com/w3c/web-platform-tests/pull/8963/files#diff-e15b270b3ab026ff3164e70694bf6aac

File: lint.whitelist [hunk 2]
```diff
@@ -66,6 +66,7 @@ TRAILING WHITESPACE, INDENT TABS, CR AT EOL: *.ani
 TRAILING WHITESPACE, INDENT TABS, CR AT EOL: *.cur
 TRAILING WHITESPACE, INDENT TABS, CR AT EOL: *.ico
 TRAILING WHITESPACE, INDENT TABS, CR AT EOL: *.wasm
+TRAILING WHITESPACE: *.sub.headers
```
Comment (yoavweiss) [inline lint.whitelist]: Just to make sure, is the trailing whitespace necessary for the test? If so, can we exclude just the tests that need it?

Comment (cvazac) [reply lint.whitelist]: Yes, the trailing whitespace is part of the test. I could whitelist just those ~3 tests, but the file names could change. 

If you prefer, we could just do:
```
TRAILING WHITESPACE: server-timing/resources/parsing/*.sub.headers
``

Comment (yoavweiss) [reply lint.whitelist]: Yeah, that's probably better

**Pull request merged**